### PR TITLE
Start to convert buildings in test scripting to Python FOCS

### PR DESCRIPTION
**Pull request by o01eg**

### CODE REVIEW

File: test-scripting/buildings/ART_BLACK_HOLE.focs.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+from common.base_prod import BUILDING_COST_MULTIPLIER
+from common.enqueue import ENQUEUE_BUILD_ONE_PER_PLANET
+
+try:
+    from focs._buildings import *
+except ModuleNotFoundError:
+    pass
```
Comment (Cjkjvfnby) [inline test-scripting/buildings/ART_BLACK_HOLE.focs.py:7]: You could skip this part for now.  If we don't have the pyi files, these imports make no sense.

**Review by Cjkjvfnby — APPROVED** [review]
Not sure what to review, the schema looks good to me.

File: test-scripting/buildings/ART_BLACK_HOLE.focs.py (hunk 1 above)
Comment (o01eg) [reply test-scripting/buildings/ART_BLACK_HOLE.focs.py:7]: It also fixes missing builtins for Python linters.

Comment (Cjkjvfnby) [reply test-scripting/buildings/ART_BLACK_HOLE.focs.py:7]: oh, indeed.

**Comment by Cjkjvfnby** [comment]
Unrelated question, could we now remove tests for species?  They are already working and we have a test that real scripts are working well?

**Comment by o01eg** [comment]
@Cjkjvfnby Those tests also confirm we don't break species parsing while making refactoring or maybe some upcoming changes.

**Pull request merged**